hathor/builder.py

```diff
@@ -26,6 +26,8 @@
 from twisted.web import server
 from twisted.web.resource import Resource
 
+from hathor.conf.get_settings import HathorSettings as DefaultHathorSettings
```

Comment:
You don't need the get_settings.

Reply:
I have rolled back that change as I found a better way to do it and it's not necessary anymore.

Reply:
Ok!